main/FatesHistoryInterfaceMod.F90

```diff
@@ -2954,6 +2966,13 @@ subroutine update_history_dyn(this,nc,nsites,sites)
          cpatch => cpatch%younger
       end do patchloop !patch loop
 
+      ! divide basal-area-weighted height by basal area to get mean
+      if ( sum(hio_ba_si_scpf(io_si,:)) .gt. tiny ) then
```

Comment:
I've never seen tiny used as a constant, ie not used as a function call. My only concern here is that not all compilers support it and perhaps not part of the fortran standard. I could be wrong but just don't know.  tiny()  or nearzero are safe though.

Reply:
correction, I see it defined here: https://github.com/NGEET/fates/blob/2c2eb561584833c9e221eea13727c9ef4d11b809/main/FatesHistoryInterfaceMod.F90#L1802
Can we get rid of tiny and use nearzero here?